i5k/celery.py

```diff
@@ -16,3 +16,6 @@
 app.config_from_object('django.conf:settings')
 app.autodiscover_tasks(lambda: settings.INSTALLED_APPS)
 
+@app.task(bind=True)
```

Comment:
Can you point me where the documentation describes the usage of this?

Reply:
http://docs.celeryproject.org/en/latest/django/first-steps-with-django.html#using-celery-with-django

Reply:
Thanks.

However, I think it is for introduction of celery and is not for the upgrade.
Similar syntax can be found older version of the documentation: http://docs.celeryproject.org/en/3.1/django/first-steps-with-django.html

I think what we do here is "For Django users and others who want to keep uppercase names" section of:
http://docs.celeryproject.org/en/v4.1.0/whatsnew-4.0.html#upgrading-from-celery-3-1